src/App.js

```diff
@@ -14,11 +17,78 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const preparedGoods = goodsFromServer.map((good, index) => ({
+  good,
+  id: index,
+}));
+
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  resetState = ({ target }) => {
+    const goods = [...target.nextElementSibling.children];
```

Comment:
```suggestion
    const goods = [...target.nextElementSibling.children];
```
я не знаю что это, но это не нужно использовать точно